### DISCUSSION TITLE
IP address filtering

### DISCUSSION THREAD

**Discussion by RoYoMi** [CrowdStrike/psfalcon#273] [Q&A] [answered]
Trying to use the filter to find all devices with a given subnet.

`Get-FalconHost -filter "local_ip:'10.6.25.57'"`
works as expected, returning my computer.

`Get-FalconHost -filter "local_ip:*'10.6.25.*'"`
returns 
[{"code":400,"message":"invalid IP address for local_ip: 10.6.25.*"}]

**Comment by bk-cs** [comment, marked as answer]
This is not specific to PSFalcon, but how the APIs work with the `local_ip` property.  You need to use `local_ip.raw` with wildcards:
```powershell
Get-FalconHost -Filter "local_ip.raw:*'10.6.25.*'"
```
Host and Host Group Management APIs - Device Filters \[ [EU-1](https://falcon.eu-1.crowdstrike.com/documentation/84/host-and-host-group-management-apis#Appendix-A-Device-filters) | [US-1](https://falcon.crowdstrike.com/documentation/84/host-and-host-group-management-apis#Appendix-A-Device-filters) | [US-2](https://falcon.us-2.crowdstrike.com/documentation/84/host-and-host-group-management-apis#Appendix-A-Device-filters) | [US-GOV-1](https://falcon.laggar.gcw.crowdstrike.com/documentation/84/host-and-host-group-management-apis#Appendix-A-Device-filters) \]

**Reply by RoYoMi** [reply]
Interesting.  That works. 

Should probably update https://github.com/CrowdStrike/psfalcon/wiki/Filtering-Results
